docs/logs/customizing-the-sdk/README.md

```diff
@@ -26,7 +26,13 @@ TODO
 
 ### AddProcessor
 
-TODO
+A [Processor](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/logging-library-sdk.md#logprocessor)
```

Comment:
I'd remove the whole section now. As it can be misleading - for example : the logrecord is currently immutable, so there is not much use for a processor.

Reply:
removed.